services/query-cmr/app/query.ts

```diff
@@ -41,27 +41,37 @@ async function querySearchAfter(
     null,
     sessionKey,
     searchAfter,
+    'umm_json',
   );
+
   const { hits } = cmrResponse;
   const newSearchAfter = cmrResponse.searchAfter;
   logger.info(`CMR Hits: ${hits}, Number of granules returned in this page: ${cmrResponse.granules.length}`);
   let totalItemsSize = 0;
   const outputItemSizes = [];
   const catalogs = cmrResponse.granules.map((granule) => {
-    const granuleSize = granule.granule_size ? parseFloat(granule.granule_size) : 0;
+    const archiveInfo = granule.umm.DataGranule?.ArchiveAndDistributionInformation;
```

Comment:
This one is important for batching to make sure we obey size constraints. Is this returning the exact same thing as before or is it possibly different?

Reply:
It should be exactly the same.